src/use-native.ts

```diff
@@ -1,21 +1,19 @@
 import { RimrafAsyncOptions, RimrafOptions } from './index.js'
-import platform from './platform.js'
 
-const version = process.env.__TESTING_RIMRAF_NODE_VERSION__ || process.version
-const versArr = version.replace(/^v/, '').split('.')
-
-/* c8 ignore start */
-const [major = 0, minor = 0] = versArr.map(v => parseInt(v, 10))
-/* c8 ignore stop */
+/* c8 ignore next */
+const [major = 0, minor = 0] = process.version
+  .replace(/^v/, '')
+  .split('.')
+  .map(v => parseInt(v, 10))
```

Comment:
@isaacs is it worth removing this version check is `use-native` now that `v14.14.0` is no longer supported by `engines`?